frontend-react/src/pages/submissions/SubmissionDetails.test.tsx

```diff
@@ -17,6 +15,9 @@ import SubmissionDetails, {
 const timeRegex: RegExp = /[0-9]{1,2}:[0-9]{1,2} [A,P]M/;
 const mockData: ActionDetailsResource = ActionDetailsResource.dummy();
 jest.mock("rest-hooks", () => ({
```

Comment:
@rest-hooks/test tries to extend from a class in rest-hooks; however, that class is explicitly removed via this mock. This breaks its ability to even run given class extends undefined is not allowed.

Reply:
Is it one of our current mocks or could we fix this with a `jest.requireActual` for the rest of the library?

Reply:
What I found was missing PollingManager - so requireActual may solve it.